### PR TITLE
eos-default-settings: Ship default container registries configuration

### PR DESCRIPTION
**Pull request by andrunko**
Source changes at https://github.com/endlessm/eos-theme/pull/354.

https://phabricator.endlessm.com/T31300

### CODE REVIEW

**Comment by andrunko** [comment]
CI is failing with a weird error that does not seem related to the changes here and I am not sure why it started complaining now:
```
make[2]: Entering directory '/build/src/eos-theme_1.4.0+dev209.2aa0bdb/_build/sub/po'
INTLTOOL_EXTRACT="/usr/bin/intltool-extract" XGETTEXT="/usr/bin/xgettext" srcdir=../../../po /usr/bin/intltool-update --gettext-package eos-theme --pot
rm -f missing notexist
srcdir=../../../po /usr/bin/intltool-update -m
The following files contain translations and are currently not in use. Please
consider adding these to the POTFILES.in file, located in the po/ directory.

_build/sub/icons/index.theme.in

If some of these files are left out on purpose then please add them to
POTFILES.skip instead of POTFILES.in. A file 'missing' containing this list
of left out files has been written in the current directory.
Please report to http://www.endlessm.com
if [ -r missing -o -r notexist ]; then \
  exit 1; \
fi
make[2]: *** [Makefile:150: check] Error 1
```

`sub/icons/index.theme.in` is already listed on `POTFILES.skip` but without the _build dir, suggestions?

**Comment by wjt** [comment]
Nuclear option: delete all the internationalisation stuff from the build system? It's not used and there are no translations.

**Review by wjt — APPROVED** [review]
Fine if the build is fixed :)

**Comment by andrunko** [comment]
Retest this please.

**Comment by andrunko** [comment]
CI is fixed with latest changes.

**Comment by andrunko** [comment]
> Nuclear option: delete all the internationalisation stuff from the build system? It's not used and there are no translations.

Done that.

> Fine if the build is fixed :)

Looks like you forgot to review https://github.com/endlessm/eos-theme/pull/354 (the source counterpart) which should land together with this one :).

File: debian/control [hunk 1]
```diff
@@ -34,7 +34,7 @@ Replaces: eos-theme (<< 1.4.0)
 Breaks: eos-theme (<< 1.4.0)
 Description: default settings for EndlessOS
  This package contains default settings for EndlessOS, including
- GSettings overrides and X11 environment defaults.
+ GSettings overrides and X11 environment defaults, among others.
```
Comment (wjt) [inline debian/control:37]: Not worth changing again, but, this is implied by "including".

**Review by wjt — APPROVED** [review]
I thought it might be possible to remove the build dependency on intltool but it seems gnome.mk assumes it's present. Harmless in any case.

**Pull request merged**